lib/Migration/Version001000Date20240328000000.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Tables\Migration;
+
+use Closure;
+use Doctrine\DBAL\Exception;
+use OCP\BackgroundJob\IJobList;
+use OCP\IDBConnection;
+use OCP\DB\ISchemaWrapper;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+use OCA\Tables\BackgroundJob\ConvertViewColumnsFormat;
+
+class Version001000Date20240328000000 extends SimpleMigrationStep {
+	public function __construct(
+		protected IDBConnection $connection,
+		protected IJobList $jobList,
+	) {
+	}
+
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 * @throws Exception
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		return null;
+	}
+
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 */
+	public function postSchemaChange(IOutput $output, Closure $schemaClosure, array $options): void {
+		// Register background job to convert column format
+		$this->jobList->add(ConvertViewColumnsFormat::class);
```

Comment:
Need to dobule check that this does not stumble on (web) upgrades due to cached autoloading info